decidim-meetings/app/views/decidim/meetings/versions/show.html.erb

```diff
@@ -1,27 +1,20 @@
-<%= turbo_frame_tag "main_frame",  src: meetings_path %>
-
-<%= turbo_frame_tag "meeting_frame" do %>
-  <% content_for :drawer_aside do %>
-    <%= cell(
-      "decidim/versions_list",
-      versioned_resource,
-      version_path: proc { |version_index| url_for(action: :show, id: version_index) },
-      i18n_scope: "decidim.meetings.versions",
-      turbo_frame: :drawer_content_frame
-    ) %>
-  <% end %>
+<% content_for :drawer_aside do %>
```

Comment:
@entantoencuanto maybe the name should be just aside? the reference to drawers will be confusing in the future

Reply:
I decided not to change the name until the definition of the new layout of the show view, because there is "aside" in other different layouts and there are drawer aside also in other branches not merged yet.

Reply:
I've added a comment